src/dotnet-scaffolding/dotnet-scaffold-aspnet/Extensions/PropertySymbolExtensions.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel.DataAnnotations;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.DotNet.Tools.Scaffold.AspNet.Extensions;
+
+internal static class PropertySymbolExtensions
+{
+    public static bool HasRequiredAttribute(this IPropertySymbol propertySymbol)
+    {
+        return propertySymbol.GetAttributes().FirstOrDefault(
```

Comment:
Would `.Any()` work here instead of `.FirstOrDefault()`?

Reply:
yep will change, used FirstOrDefault to get and debug over the results and then had it left over.